apps/backend/src/schema/schema.ts

```diff
@@ -0,0 +1,47 @@
+import { z } from 'zod';
+
+const addressSchema = z.object({
+    address: z.string(),
+    city: z.string(),
+    state: z.string(),
+    zipcode: z.number(),
+})
+
+// Part of form to be filled out by the child's parent/legal guardian
+const guardianFormSchema = z.object({
+    childsName: z.string(),
+    ssn: z.number(),
+    dob: z.string(),
```

Comment:
Instead of string, date might be better defined as its own schema with 3 elements - month, day, year, all numbers with restrictions on minimum and maximum values and then you can construct a Date instance and have [zod validate it](https://zod.dev/?id=dates). After that, we could reformat the date in a standard way (e.g. MM/DD/YYYY).

Reply:
I think using the z.date() function should be enough to validate all date types passed in?